sshd-core/src/main/java/org/apache/sshd/common/forward/DefaultForwardingFilter.java

```diff
@@ -964,7 +965,7 @@ protected void invokePortEventListenerSignallerHolders(
 
     @Override
     protected synchronized Closeable getInnerCloseable() {
-        return builder().parallel(toString(), dynamicLocal.values()).close(acceptor).build();
+        return builder().parallel(toString(), dynamicLocal.values()).close(dynamicAcceptor).build();
```

Comment:
Don't we need to close the local acceptor as well ?

Reply:
It is possible, but honestly, I don't know the sshd code enough and I just let myself be guided by that "dynamicLocal.values()"

Reply:
Yes, both would have to be closed.

Reply:
Well, but... I don't known how to do... '':-)

Reply:
```
        return builder().parallel(toString(), dynamicLocal.values())
                .close(localAcceptor).close(dynamicAcceptor).build();
```

Reply:
A lot of thanks!